infra/aws/terraform/prow-build-cluster/modules/iam/users.tf

```
@@ -0,0 +1,22 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+data "aws_iam_user" "user_xmudrii" {
+  user_name = "xmudrii"
+}
+data "aws_iam_user" "user_pprzekwa" {
+  user_name = "pprzekwa"
+}
```

I was thinking about sth like this:

```suggestion
data "aws_iam_user" "eks_admins" {
  count = length(var.eks_admins)
  user_name = element(var.eks_admins, count.index)
}
```

---

Good point, fixed.